homeassistant/components/samsungtv/media_player.py

```diff
@@ -224,14 +224,25 @@ async def async_update(self) -> None:
         self._update_from_upnp()
 
     @callback
-    def _update_from_upnp(self) -> None:
+    def _update_from_upnp(self) -> bool:
         if (dmr_device := self._dmr_device) is None:
```

Comment:
Maybe a comment that the upnp events can affect other attributes that we currently do not track so we want to avoid checking every attribute in via `async_write_ha_state` when we only care about 2 of them currently?